user/search.go

```diff
@@ -22,6 +22,10 @@ type SearchOptions struct {
 	Omit []string `json:"o,omitempty"`
 
 	Limit int `json:"-"`
+	//CMValue is matched against the user's contact method phone number.
```

Comment:
Blank line before the comment line, and space before the text
```suggestion
	Limit int `json:"-"`

	// CMValue is matched against the user's contact method phone number.
```